tico/passes/legalize_causal_mask_value.py

```diff
@@ -60,21 +61,29 @@ def call(self, exported_program: ExportedProgram) -> PassResult:
             input = args.input
             other = args.other
 
-            if (
-                isinstance(input, torch.fx.Node)
-                and input.name
-                in exported_program.graph_signature.lifted_tensor_constants
-            ):
+            def is_lifted_const_node(n: torch.fx.Node):
+                if not isinstance(n, torch.fx.Node):
+                    return False
+
+                is_lifted_tensor = (
+                    n.name in exported_program.graph_signature.lifted_tensor_constants
+                )
+                is_lifted_input_tensor = (
+                    n.name
+                    in exported_program.graph_signature.inputs_to_lifted_tensor_constants
+                )
```

Comment:
AFAIK, just checking the `inputs_to_lifted_tensor_constants` is enough. Or you can use `is_lifted_tensor_constant` api of torch.